game/tests/utils/teacher.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Code for Life
+#
+# Copyright (C) 2016, Ocado Innovation Limited
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# ADDITIONAL TERMS – Section 7 GNU General Public Licence
+#
+# This licence does not grant any right, title or interest in any “Ocado” logos,
+# trade names or the trademark “Ocado” or any other trademarks or domain names
+# owned by Ocado Innovation Limited or the Ocado group of companies or any other
+# distinctive brand features of “Ocado” as may be secured from time to time. You
+# must not distribute any modification of this program using the trademark
+# “Ocado” or claim any affiliation or association with Ocado or its employees.
+#
+# You are not authorised to use the name Ocado (or any of its trade names) or
+# the names of any author or contributor in advertising or for publicity purposes
+# pertaining to the distribution of this program, without the prior written
+# authorisation of Ocado.
+#
+# Any propagation, distribution or conveyance of this program must include this
+# copyright notice and these terms. You must not misrepresent the origins of this
+# program; modified versions of the program must be marked as such and not
+# identified as the original program.
+import random
+import string
+from django.core import mail
+import sys
+import email
+
+from portal.models import Teacher, School
+from portal.helpers.email import generate_token
+
+
+def generate_details(**kwargs):
+    title = kwargs.get('title', 'Mr')
+    first_name = kwargs.get('first_name', 'Test')
+    last_name = kwargs.get('last_name', 'Teacher')
+    email_address = kwargs.get('email_address', 'testteacher%d@codeforlife.com' % random.randint(1, sys.maxint))
+    password = kwargs.get('password', 'Password1')
+
+    return title, first_name, last_name, email_address, password
```

Comment:
Returning a dict here would make more sense as then it can be used as `teacher = Teacher(**generate_details())`.

E.g. something like this:
```python
def generate_details(**kwargs):
    result = {
        'title': 'Mr',
        'first_name': 'Test',
        'last_name': 'Teacher',
        'email_address': 'testteacher%d@codeforlife.com' % random.randint(1, sys.maxint),
        'password': 'Password1',
    }
    result.update(kwargs)
    return result
```